examples/Feeds/RemoveFlightsFeedItemAttributeValue.cs

```diff
@@ -0,0 +1,307 @@
+    /// The attribute you are removing must be present on the feed.
+    ///
+    /// This example is specifically for removing an attribute of a flights feed item,
+    /// but it can also be changed to work with other feed types.
+    ///
+    /// To make this work with other feed types, replace the FlightPlaceholderField enum with the
+    /// equivalent one of your feed type, and replace the appropriate attribute names in the
+    /// GetFeeds method.
+    /// </summary>
+    public class RemoveFlightsFeedItemAttributeValue : ExampleBase
+    {
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            RemoveFlightsFeedItemAttributeValue codeExample =
+                new RemoveFlightsFeedItemAttributeValue();
+            Console.WriteLine(codeExample.Description);
+
+            // The Google Ads customer ID for which the call is made.
+            long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+
+            // The Feed ID to which the feed items belong.
+            long feedId = long.Parse("INSERT_FEED_ID_HERE");
+
+            // The ID of the feed items to be updated.
+            long feedItemId = long.Parse("INSERT_FEED_ITEM_ID_HERE");
+
+            // The flight placeholder field name for the attribute to be removed.
+            string flightPlaceholderFieldName = "INSERT_FLIGHT_PLACEHOLDER_FIELD_NAME_HERE";
+
+            codeExample.Run(new GoogleAdsClient(), customerId, feedId, feedItemId,
+                flightPlaceholderFieldName);
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description => "Removes feed items from a feed.";
+
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="client">The Google Ads API client.</param>
+        /// <param name="customerId">The Google Ads customer ID for which the call is made.</param>
+        /// <param name="feedId">The Feed ID to which the feed items belong.</param>
+        /// <param name="feedItemId">The ID of the feed items to be updated.</param>
+        /// <param name="flightPlaceholderFieldName">The flight placeholder field name for the
+        ///     attribute to be removed.</param>
+        public void Run(GoogleAdsClient client, long customerId, long feedId, long feedItemId,
+            string flightPlaceholderFieldName)
+        {
+            // Creates the Feed Item service client.
+            FeedItemServiceClient feedItemServiceClient = client.GetService(
+                Services.V4.FeedItemService);
+
+            // Strip any special characters and whitespace from the flight placeholder field name.
+            Regex specialCharsAndWhitespaceRegex = new Regex("[*'\",_&#^@\\s]");
```

Comment:
Normally, special chars isn't an issue, even in the html world -- ASP frameworks ensure they all convert back and forth between &nbsp; to space, for instance. So we only need to do flightPlaceholderFieldName.Trim() instead of using Regex.

Is this regex added here because it was there in the golden, or did you anticipate / notice an issue that made this regex necessary?

Reply:
Removed per our conversation.